tests/test_forcing.py

```diff
@@ -0,0 +1,1025 @@
+from esmvalcore.experimental import get_recipe
+import deepdiff
+import copy
+import pytest
+from ewatercycle.forcing import update_marrmot, update_lisflood, update_hype, update_wflow, update_pcrglobwb
+
+TEST_INPUT_MARRMOT = (
+    {},
+    {
+        'startyear': 1990,
+        'endyear': 2018,
+        'forcings': ['ERA-Interim', 'ERA5'],
+        'shapefile': 'Meuse/Meuse.shp',
```

Comment:
Where do these shapefiles come from? They're not part of the package, are they?

Reply:
These are the default values in the recipes on `esmvaltool`. I use these values to see if the original recipes can be reproduced.

Reply:
ah okay check